backend/app/services/chat/db_handler.py

```
@@ -126,13 +163,71 @@ def _update_task_status_sync(self, task_id: int) -> None:
+                    loop = asyncio.get_event_loop()
+                    if loop.is_running():
+                        asyncio.run_coroutine_threadsafe(
+                            emit_task_status_update(user_id, task_id, status, progress),
+                            loop,
+                        )
+                        logger.debug(
+                            f"[WS] Scheduled task:status via fallback loop for task={task_id}"
+                        )
+                    else:
+                        logger.warning(
+                            f"Could not emit task:status event - no running event loop available"
```

Ruff (0.14.8)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)